splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceAdmin.java

```diff
@@ -1317,10 +1323,9 @@ public static void SYSCS_EMPTY_GLOBAL_STATEMENT_CACHE() throws SQLException{
         }
 
         for (HostAndPort server : servers) {
-            try (Connection connection = RemoteUser.getConnection(server.toString())) {
-                try (PreparedStatement ps = connection.prepareStatement("call SYSCS_UTIL.SYSCS_EMPTY_STATEMENT_CACHE()")) {
-                    ps.execute();
-                }
+            try (Connection connection = RemoteUser.getConnection(server.toString());
+                 Statement s = connection.createStatement()) {
```

Comment:
I think you should wrap the statement with `try-with-resource`

Reply:
the statement IS wrapped in a try-with-resource, check the brackets, note that you can also have multiple statements in a try-with-resources, e.g.

```
try( MyClass1 c1 = new MyClass1();  MyClass2 = new MyClass2(c1); )
{
 ...
}
```

Reply:
I didn't notice the semicolon, my bad.